### PR TITLE
Correcting Line 22

### PR DESCRIPTION
**Pull request by Mohit-Nain**
Menu drop down is correct description for the control , rather than the menu that drops down

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/auth/google.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ To enable the Google OAuth2 you must register your application with Google. Goog
 First, you need to create a Google OAuth Client:
 
 1. Go to https://console.developers.google.com/apis/credentials
-2. Click the 'Create Credentials' button, then click 'OAuth Client ID' in the menu that drops down
+2. Click the 'Create Credentials' button, then click 'OAuth Client ID' in the drop down menu
```
Comment (oddlittlebird) [inline docs/sources/auth/google.md]:
```suggestion
2. Click **Create Credentials**, then click **OAuth Client ID** in the drop-down menu
```

Comment (oddlittlebird) [reply docs/sources/auth/google.md]: Per Microsoft Style Guide, most clickable elements in the UI are bold. 

Also, see their guidance on the use of "drop-down." https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/drop-down

File: docs/sources/auth/google.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ To enable the Google OAuth2 you must register your application with Google. Goog
 First, you need to create a Google OAuth Client:
 
 1. Go to https://console.developers.google.com/apis/credentials
```
Comment (oddlittlebird) [inline docs/sources/auth/google.md]:
```suggestion
1. Go to https://console.developers.google.com/apis/credentials.
```

Comment (Mohit-Nain) [reply docs/sources/auth/google.md]: Thank you for the suggestion, please merge

**Comment by Mohit-Nain** [comment]
Suggestions have been incorporated, please merge

**Review by daniellee — APPROVED** [review]
Looks good to me

**Pull request merged**